packages/components/src/components/Alert/Alert.test.tsx

```diff
@@ -101,12 +102,12 @@ describe('Alert', () => {
         render(<Alert variant="info" {...commonProps} />)
 
         expect(screen.root).toHaveStyle({
-          backgroundColor: '#e1f3f8',
-          borderLeftColor: '#28a0cb',
+          backgroundColor: Colors.vadsColorFeedbackSurfaceInfoOnLight,
```

Comment:
@narin and I had discussed this before--not sure we want to update any of the unit tests with this ticket (unless the color changed and a new hex is needed to pass). 

Not sure we hammered it down, but know we had floated either sticking with hex's or using primitive tokens in the tests. Either way, we didn't want to use the exact same tokens in the unit tests since that's not really validating anything other than they weren't changed in the component without the unit test also being changed (not a helpful test). Such a circular test would not catch if changes occur in the tokens (including from the VADS side) because the test would automatically update in the same way while we'd want it to fail to confirm that a change occurred and it should be manually validated it was as expected and not someone made an oopsie updating a token incorrectly which could compromise the appearance of our components.

Reply:
Thanks. Narin helped me revert the unit test files back to main and I updated the hex codes as needed. I also updated the AC in the original ticket.